README.md

```diff
@@ -15,84 +15,20 @@
 [![QAequilibraE artifacts](https://github.com/AequilibraE/aequilibrae/actions/workflows/build_artifacts_qgis.yml/badge.svg)](https://github.com/AequilibraE/aequilibrae/actions/workflows/build_artifacts_qgis.yml)
 
 
-AequilibraE is the first comprehensive Python package for transportation modeling. It aims to provide all the
+AequilibraE is a fully-featured Open-Source transportation modeling package and the first comprehensive package
+of its king for the Python ecosystem. It provides all the most fundamental
 resources not available from other open-source packages in the Python (NumPy, really) ecosystem.
 
 ## Comprehensive documentation
 
 [AequilibraE documentation built with Sphinx ](http://www.aequilibrae.com)
 
-## What is available
-
-* Importing networks from OSM
-* Synthetic gravity/IPF
-* Traffic assignment (All-or Nothing, MSA, Frank-Wolfe, Conjugate Frank-Wolfe & Biconjugate-FrankWolfe)
-* Network Skimming & node-to-node path computation
-* Fast Matrix format based on NumPy
-* GTFS Import
 
 ### What is available only in QGIS
 
 Some common resources for transportation modeling are inherently visual, and therefore they make more sense if
 available within a GIS platform. For that reason, many resources are available only from AequilibraE's 
 [QGIS plugin](http://plugins.qgis.org/plugins/qaequilibrae/),
 which uses AequilibraE as its computational workhorse and also provides GUIs for most of AequilibraE's tools. Said tool
-is developed independently, although in parallel, and more details can be found in its 
+is developed independently and a little delayed with relationship to the Python package, and more details can be found in its 
 [GitHub repository](https://github.com/AequilibraE/qaequilibrae).
-
-
-### What is not planned to be available any time soon
```

Comment:
Is this not worth keeping?

Reply:
These things fall through the cracks... Until the other day, PT assignment was in the list...    I think that a proper roadmap will solve this issue.